lib/blobs/usbboot/README.md

```diff
@@ -1,5 +0,0 @@
-usbboot
-=======
-
-The files in this directory were taken from
```

Comment:
Might be worth bringing this file back too, to clarify which files are covered by which license?

Reply:
I divided the files in subdirectories with their own licenses. Do you think that's enough?

Reply:
Well having the files in separate directories makes it harder to go back to `rpiboot` for testing purposes, but I guess that's not really much of a problem.